scm/driver/github/content.go

```diff
@@ -55,20 +76,19 @@ type content struct {
 	Type    string `json:"type"`
 }
 
-type contentUpdate struct {
-	Sha     string `json:"sha"`
-	Message string `json:"message"`
-	HTMLURL string `json:"html_url"`
-	Author  struct {
-		Name  string    `json:"name"`
-		Email string    `json:"email"`
-		Date  time.Time `json:"date"`
-	} `json:"author"`
-	Committer struct {
-		Name  string    `json:"name"`
-		Email string    `json:"email"`
-		Date  time.Time `json:"date"`
-	} `json:"committer"`
```

Comment:
why where the `json:...` tags removed?  these are required to properly marshal and unmarshal data to json.  Have you tested your code to make sure it works?